DATA/production/configurations/2022/LHC22f/apass1/async_pass.sh

```diff
@@ -169,11 +169,34 @@ fi
 
 echo "[INFO (async_pass.sh)] envvars were set to TFDELAYSECONDS ${TFDELAYSECONDS} TIMEFRAME_RATE_LIMIT ${TIMEFRAME_RATE_LIMIT}"
 
+# root output enabled only for 2 permille of the cases
+# keeping AO2D.root QC.root o2calib_tof.root MFTAssessment.root mchtracks.root mchclusters.root
+SETTING_ROOT_OUTPUT="ENABLE_ROOT_OUTPUT_o2_mch_reco_workflow= ENABLE_ROOT_OUTPUT_o2_mft_reco_workflow= ENABLE_ROOT_OUTPUT_o2_tof_matcher_workflow= ENABLE_ROOT_OUTPUT_o2_aod_producer_workflow= ENABLE_ROOT_OUTPUT_o2_qc= "
+
+if [[ -z $RND_PERMILLE ]]; then
+  RND_PERMILLE=$((1 + $RANDOM % 2000))
+fi
+echo "Randomly generated number for current reco = $RND_PERMILLE"
+if [[ $RND_PERMILLE -ge 2 ]]; then
```

Comment:
Don't you set `RND_PERMILLE` to something between 1 and 2000 above?
```suggestion
if [[ $RND_PERMILLE -ge 1000 ]]; then
```

Reply:
Ah nevermind, you want to store only one permille...